update_prevalence.py

```diff
@@ -573,6 +576,8 @@ def parse_json(cache: DataCache, model: Type[Model], url: str) -> List[Model]:
 
 
 def ignore_jhu_place(line: JHUCommonFields) -> bool:
+    if line.Combined_Key == "":
+        return True
```

Comment:
This was one of the main things that got it working. Does this seem like a good addition to you?

Reply:
I'd want to know which lines you're seeing that have an empty Combined_Key, but yes, seems like a fine workaround in principle.

Reply:
![image](https://user-images.githubusercontent.com/32290/98448712-faf52680-20e2-11eb-8237-d6c40c32ebf0.png)

I get this when I run it without that line. It just appears to be a blank line as far as I can tell.

Let me know if you'd like any more info on this change before we accept it.

Reply:
Nah, that's enough for me!